common/src/main/java/hardcorequesting/common/client/interfaces/edit/PickMobMenu.java

```diff
@@ -0,0 +1,235 @@
+package hardcorequesting.common.client.interfaces.edit;
+
+import com.google.common.collect.ImmutableList;
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.blaze3d.vertex.PoseStack;
+import hardcorequesting.common.client.interfaces.*;
+import hardcorequesting.common.quests.task.icon.TameMobsTask;
+import hardcorequesting.common.util.Translator;
+import net.minecraft.client.resources.language.I18n;
+import net.minecraft.core.Registry;
+import net.minecraft.network.chat.FormattedText;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.EntityType;
+import net.minecraft.world.entity.player.Player;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class PickMobMenu extends GuiEditMenuExtended {
+    //TODO add support for entity tags to replace functionality of this special case
```

Comment:
Do you plan to do this or is this TODO for a future contributor?

Reply:
It's meant as a TODO for a future contributor. I had decided that entity tags was out of scope for this pull request.